diff --git a/dist/resources/helpers/declaration-title.js b/dist/resources/helpers/declaration-title.js
index c8529a5bd97140e30d532321b0ca26749bd47d68..da2fe6a1338bd556423a09e5c11daad9a3f05aa4 100644
--- a/dist/resources/helpers/declaration-title.js
+++ b/dist/resources/helpers/declaration-title.js
@@ -20,7 +20,7 @@ function default_1(theme) {
         }
         md.push(`${this.flags.isRest ? '... ' : ''} **${(0, utils_1.escapeChars)(this.name)}**`);
         if (this instanceof typedoc_1.DeclarationReflection && this.typeParameters) {
-            md.push(`<${this.typeParameters
+            md.push(`\\<${this.typeParameters
                 .map((typeParameter) => `\`${typeParameter.name}\``)
                 .join(', ')}\\>`);
         }
diff --git a/dist/resources/helpers/reflection-title.js b/dist/resources/helpers/reflection-title.js
index 24bbd50a94f7e389406a07d0537601fe90fc93f7..79f5cee8c7786742182699bb455d396639448694 100644
--- a/dist/resources/helpers/reflection-title.js
+++ b/dist/resources/helpers/reflection-title.js
@@ -5,11 +5,6 @@ const utils_1 = require("../../utils");
 function default_1(theme) {
     Handlebars.registerHelper('reflectionTitle', function (shouldEscape = true) {
         const title = [''];
-        if (this.model &&
-            this.model.kindString &&
-            this.url !== this.project.url) {
-            title.push(`${this.model.kindString}: `);
-        }
         if (this.url === this.project.url) {
             title.push(theme.indexTitle || this.project.name);
         }
@@ -19,7 +14,7 @@ function default_1(theme) {
                 const typeParameters = this.model.typeParameters
                     .map((typeParameter) => typeParameter.name)
                     .join(', ');
-                title.push(`<${typeParameters}${shouldEscape ? '\\>' : '>'}`);
+                title.push(`\\<${typeParameters}${shouldEscape ? '\\>' : '>'}`);
             }
         }
         return title.join('');
diff --git a/dist/resources/helpers/signature-title.js b/dist/resources/helpers/signature-title.js
index 38ade0bf586ad0c1bea30140b263c7e359cb5d1d..99891db19aeb181d92f661804b619edc9d0130d5 100644
--- a/dist/resources/helpers/signature-title.js
+++ b/dist/resources/helpers/signature-title.js
@@ -20,7 +20,7 @@ function default_1(theme) {
             md.push(`**${this.name}**`);
         }
         if (this.typeParameters) {
-            md.push(`<${this.typeParameters
+            md.push(`\\<${this.typeParameters
                 .map((typeParameter) => `\`${typeParameter.name}\``)
                 .join(', ')}\\>`);
         }
diff --git a/dist/resources/helpers/type.js b/dist/resources/helpers/type.js
index 8569286f2dabb1dff0ea8f7698111a73dfac27ae..1d3cfadcf08db32ed60b9f9a7e16bda2334775a1 100644
--- a/dist/resources/helpers/type.js
+++ b/dist/resources/helpers/type.js
@@ -99,7 +99,7 @@ function getDeclarationType(model) {
 function getFunctionType(modelSignatures) {
     const functions = modelSignatures.map((fn) => {
         const typeParams = fn.typeParameters
-            ? `<${fn.typeParameters
+            ? `\\<${fn.typeParameters
                 .map((typeParameter) => typeParameter.name)
                 .join(', ')}\\>`
             : [];
@@ -127,7 +127,7 @@ function getReferenceType(model, emphasis) {
                 : `\`${model.name}\``);
         }
         if (model.typeArguments && model.typeArguments.length > 0) {
-            reflection.push(`<${model.typeArguments
+            reflection.push(`\\<${model.typeArguments
                 .map((typeArgument) => Handlebars.helpers.type.call(typeArgument))
                 .join(', ')}\\>`);
         }